index.html

```diff
@@ -1251,37 +1251,37 @@
 
 </aside>
 
-### Ancillary uses
+### Supporting uses
 
 [=Sites=] sometimes use data in ways that aren't needed for the user's immediate
```

Comment:
Suggested by @wseltzer for a separate PR:
```suggestion
[=Sites=] sometimes use data in ways that aren't directly needed for the user's immediate
```

Reply:
Sent as https://github.com/w3ctag/privacy-principles/pull/444.